todolist/style.css

```
@@ -36,36 +38,112 @@ header a {
   background-color: #03f4fc;
   border: 0;
   border-radius: 7px;
-  height: 30px;
+  height: 60%;
   width: 100%;
+  font-size: 18px;
 }
 
 #input-text button {
-  height: 100%;
+  height: 40%;
   width: 100%;
   border: 1px solid #dae1e6;
   background-color: #03f4fc;
 }
-#textArea h2 {
+
+#text-area h2 {
   margin-bottom: 20px;
 }
 
-#textArea span {
+#text-area span {
   position: relative;
   display: block;
   border-bottom: 1px solid #dae1e6;
   overflow: hidden;
-  margin: 15px 0px;
+  padding: 15px 0px;
   font-size: 18px;
 }
 
-#textArea button {
-  position: absolute;
-  right: 0;
-  border: 0;
+#text-area span:hover,
+#text-area span:focus {
+  cursor: pointer;
 }
 
 .done {
   text-decoration: line-through;
   color: gray;
 }
+
+.modal {
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
```

```suggestion
  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
```

CSS는 같은 속성을 2개 이상 사용하지 않습니다.